web_sdk/BUILD.gn

```diff
@@ -547,6 +547,8 @@ if (!is_fuchsia) {
   zip_bundle_from_file("flutter_web_sdk_archive") {
     if (target_os == "wasm") {
       output = "flutter-web-sdk.zip"
+    } else if (is_win) {
+      output = "flutter-web-sdk-${full_target_platform_name}.zip"
```

Comment:
cc @eyebrowsoffire, needed because of this: https://github.com/flutter/engine/pull/38394#issuecomment-1376449832

Reply:
Bleh, at some point I'd really like to make it so that we just produce a single `flutter-web-sdk.zip` artifact to the dart infra bucket and make it so that the flutter tool always consumes that artifact, regardless of the host platform (all these `flutter-web-sdk-{target}.zip` artifacts are actually completely identical anyway). But this is fine for now.